src/libraries/System.IO.FileSystem.Watcher/tests/Args.FileSystemEventArgs.cs

```diff
@@ -8,86 +8,93 @@ namespace System.IO.Tests
     public class FileSystemEventArgsTests
     {
         [Theory]
-        [PlatformSpecific(TestPlatforms.Windows)]
-        [InlineData(WatcherChangeTypes.Changed, "D:\\", "foo.txt", "D:\\foo.txt")]
-        [InlineData(WatcherChangeTypes.Changed, "E:\\bar", "foo.txt", "E:\\bar\\foo.txt")]
-        [InlineData(WatcherChangeTypes.All, "D:\\", "foo.txt", "D:\\foo.txt")]
-        [InlineData(WatcherChangeTypes.All, "E:\\bar", "foo.txt", "E:\\bar\\foo.txt")]
-        public static void FileSystemEventArgs_ctor_AbsolutePaths(WatcherChangeTypes changeType, string directory, string name, string expectedFullPath)
+        [InlineData(WatcherChangeTypes.Changed, "C:\\bar", "foo.txt")]
+        [InlineData(WatcherChangeTypes.All, "C:\\bar", "foo.txt")]
+        [InlineData((WatcherChangeTypes)0, "C:\\bar", "")]
+        [InlineData((WatcherChangeTypes)0, "C:\\bar", null)]
```

Comment:
Unix will treat C:\ as a relative path; better use something that is relative in both OSes. 
```suggestion
        [InlineData(WatcherChangeTypes.Changed, "bar", "foo.txt")]
        [InlineData(WatcherChangeTypes.All, "bar", "foo.txt")]
        [InlineData((WatcherChangeTypes)0, "bar", "")]
        [InlineData((WatcherChangeTypes)0, "bar", null)]
```

Reply:
fixed